index.bs

```diff
@@ -6368,9 +6368,10 @@ character after an existing type.
 The [=nullable types/inner type=] must not be:
 
 *   {{any}},
-*   a [=Promise type=],
+*   a [=promise type=],
+*   an [=observable array type=],
 *   another nullable type, or
-*   a [=union type=] that itself [=includes a nullable type=]
+*   an [=union type=] that itself [=includes a nullable type=]
```

Comment:
Nit: this should stay "a", see https://www.writersdigest.com/write-better-fiction/a-before-consonants-and-an-before-vowels-is-not-the-rule